src/components/ProgressSpinner/ProgressSpinner.module.scss

```diff
@@ -0,0 +1,45 @@
+.notAnimated,
+.animated {
+    display: flex;
+    justify-content: space-around;
+}
+
+.animated {
+    animation: uploaded 1s ease-in-out;
+    animation-fill-mode: forwards;
+}
+
+@keyframes uploaded {
+    from {
+        opacity: 1;
+    }
+
+    to {
+        opacity: 0;
+        transform: scale(2.5);
+    }
+}
+
+.circle {
+    position: absolute;
+    width: 200px;
```

Comment:
I suggest using rem or rm over px.

Reply:
I used px here so it matches the value of the SVG.
Using rem or em will mess up the design of the entire "spinner".
Even if I put the value in rem, it will not be responsive.
I checked the design on a mobile phone, it looks good.

Reply:
Ah, yes you right on that one.